scalikejdbc-streams/README.md

```diff
@@ -0,0 +1,54 @@
+# ScalikeJDBC Streams
+
+See the website.
+
+http://scalikejdbc.org/
+
+It is assumed to be used in batch application.
```

Comment:
The description should explain what the module is first. How about doing like this?

```
scalikejdbc-streams is a Reactive Streams 1.0 compliant implementation.

The library is originally assumed to be suitable for batch applications.
You need to keep borrowing a connection while someone is subscribing a stream provided by scalikejdbc-streams.
Be aware of the max size of connection pool and carefully monitor the state of the pool.
```